cmd/iot-transform-fiware/main.go

```diff
@@ -84,3 +66,72 @@ func setupRouterAndWaitForConnections(logger zerolog.Logger) {
 		logger.Fatal().Err(err).Msg("failed to start router")
 	}
 }
+
+func NewMeasurementTopicMessageHandler(messenger messaging.MsgContext, contextBrokerClientUrl string) messaging.TopicMessageHandler {
+	transformerRegistry := registry.NewTransformerRegistry()
+
+	return func(ctx context.Context, msg amqp.Delivery, logger zerolog.Logger) {
+		ctx = logging.NewContextWithLogger(ctx, logger)
+		logger.Info().Str("body", string(msg.Body)).Msg("received message")
+
+		messageAccepted := iotCore.MessageAccepted{}
+
+		err := json.Unmarshal(msg.Body, &messageAccepted)
+		if err != nil {
+			logger.Error().Err(err).Msg("unable to unmarshal incoming message")
+			return
+		}
+
+		contextBrokerClient := client.NewContextBrokerClient(contextBrokerClientUrl, client.Tenant(messageAccepted.Tenant()))
+		measurementType := measurements.GetMeasurementType(messageAccepted)
+
```

Comment:
Sätt measurement_type som Str property på loggern redan här, så slipper du formatera in det på tre olika ställen längre ned.